File: print_product.py
# Usage: print_product.py num1 num2 outfile this is the unicorn branch
import sys
num1=float(sys.argv[1])
num2=float(sys.argv[2])
fout=open(sys.argv[3],’w’)
fout.write(str(num1*num2)+’\n’)

File: print_product.jl
# Usage: print_product.jl num1 num2 outfile
num1=parse(Float64,ARGS[1])
num2=parse(Float64,ARGS[2]) fname=ARGS[3]
open(fname, "w") do f
  write(f, string(num1*num2,'\n'))
end
